packaging/rpm/microshift-airgap.spec

```diff
@@ -0,0 +1,31 @@
+Name: microshift
```

Comment:
@fzdarsky was asking me if we could call this microshift-container-images.spec instead.

Reply:
Resp. just "microshift-images.{spec,rpm}", yes. So users would either just `dnf install microshift` and pull images over the network or `dnf install microshift microshift-images` and have images ready on the system when microshift starts.